lib/auth/auth.js

```diff
@@ -212,6 +212,7 @@ module.exports = {
     },
     client: {
         generateV4Headers,
+        constructStringToSignV2,
```

Comment:
Don't you also need `constructStringToSignV4`?

Reply:
Don't think so. The Google Storage backend only supports v2 signing at the moment.

Reply:
@alexanderchan-scality I though GCP also supported V4 but we don't need to patch the signing module?